docs/G-Codes.md

```diff
@@ -291,3 +291,16 @@ enabled.
   - `RESPOND TYPE=error MSG="<message>"`: echo the message prepended with `!! `.
   - `RESPOND PREFIX=<prefix> MSG="<message>"`: echo the message prepended with `<prefix>`
     (The `PREFIX` parameter will take priority over the `TYPE` parameter)
+
+## Pause Resume
+
+The following commands are available when the "pause_resume" config section
+is enabled:
+  - `PAUSE [CAPTURE=<TRUE | FALSE>]`:  Pauses the current print.  If CAPTURE
```

Comment:
Just out of curiosity, is there a gain to conditionally capturing the position?  (Would there be any harm in unconditionally capturing the position, but only conditionally restoring it?)

Reply:
No, no harm in doing so.  Perhaps it is best to remove the option, I can't foresee any reason someone would want to pause, move the tool, then resume the print without moving back.  I will remove the option and make it required.